src/CowAmmModule.sol

```diff
@@ -97,21 +97,16 @@ contract CowAmmModule {
      * @param _composableCow The address of `ComposableCoW`.
      * @param _handler The address of the ConstantProduct AMM implementation for creating new CoW AMMs.
      */
-    constructor(
-        GPv2Settlement _settler,
-        ExtensibleFallbackHandler _extensibleFallbackHandler,
-        ComposableCoW _composableCow,
-        IConditionalOrder _handler
-    ) {
+    constructor(address _settler, address _extensibleFallbackHandler, address _composableCow, address _handler) {
```

Comment:
I liked the explicit typing because it made it harder to mix up different addresses in the constructor arguments.